shell/ev-window.c

```diff
@@ -5334,7 +5328,8 @@ view_menu_link_popup (EvWindow *ev_window,
             switch (ev_link_action_get_action_type (ev_action)) {
             case EV_LINK_ACTION_TYPE_GOTO_DEST:
             case EV_LINK_ACTION_TYPE_GOTO_REMOTE:
-                show_internal = TRUE;
+                //  2021-03-21 WHY WAS THIS BEING SET BUT NEVER USED? 
+                //show_internal = TRUE;
```

Comment:
Need to look further into if this is variable no longer needed or just wasn't being used properly.